VTEX - Pick and Pack Protocol API.json

```
@@ -0,0 +1,6820 @@
+                        "url": "https://api.pickingnpacking.com/{environment}/v1/tracking/hook",
+                        "variables": {
+                            "environment": {
+                                "default": "DefaultParameterValue"
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "components": {
+        "parameters": {
+            "Content-Type": {
+                "name": "Content-Type",
+                "in": "header",
+                "description": "Type of the content being sent.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "application/json"
+                }
+            },
+            "Accept": {
+                "name": "Accept",
+                "in": "header",
+                "description": "HTTP Client Negotiation _Accept_ Header. Indicates the types of responses the client can understand.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "application/json"
+                }
+            },
+            "Authorization": {
+                "name": "Authorization",
+                "in": "header",
+                "description": "Authentication JWT.",
+                "required": true,
+                "style": "simple",
+                "schema": {
+                    "type": "string",
+                    "example": "{token}"
+                }
+            }
+        },
+        "schemas": {
+            "CreatetokenRequest": {
+                "required": [
+                    "apiKey"
+                ],
+                "type": "object",
+                "properties": {
+                    "apiKey": {
+                        "type": "string",
+                        "description": "The`API KEY` previously generated by the Pick and Pack team."
+                    }
+                },
+                "example": {
+                    "apiKey": "{pnpApiKey}"
+                }
+            },
+            "ServiceBody": {
+                "required": [
+                    "message",
+                    "data"
+                ],
+                "type": "object",
+                "properties": {
+                    "message": {
+                        "type": "string",
+                        "description": "Status message about the request."
+                    },
+                    "data": {
+                        "$ref": "#/components/schemas/Data",
+                        "type": "object",
+                        "description": "Service general information."
+                    }
+                }
+            },
+            "Data": {
+                "required": [
+                    "id",
+                    "orderId",
+                 
```

```suggestion
                        "description": "Delivery shipping price."
```suggestion
                        "description": "Delivery shipping price in cents."

```
```